core/backend/src/test/integration/IModelWrite.test.ts

```diff
@@ -1068,8 +1069,72 @@ describe("IModelWriteTest (#integration)", () => {
     schemaLock = await IModelHost.hubAccess.querySchemaLock({ requestContext: superRequestContext, briefcase: superBriefcaseProps });
     assert.isFalse(schemaLock); // Validate no schema locks held by the hub
 
-    /* Cleanup after test */
     await IModelHost.hubAccess.deleteIModel({ requestContext: managerRequestContext, contextId: projectId, iModelId });
   });
 
+  it("create SpatialViewDefinition and throw errors on bad input (#integration)", async () => {

```

Comment:
This test is not an integration test. It should be in standalone tests. See Category.test.ts

Reply:
Moved test to the new location.

Reply:
I have moved it to the new location now - this is definitely a standalone test and not an integration test.